SOURCES/src/cdlink/CDLINK.ASM

```diff
@@ -3539,7 +3539,9 @@ DemoCheck proc	near
 	mov	[wDemoDiff],ax
 	cmp	ax,32760		; 18.2*30*60
 DemoJump:				; nop this instruction if not demo
-	jae	DemoExpired
+	nop
+	nop
+;	jae	DemoExpired

```

Comment:
I presume the NOPs are to maintain relative position in the code, is that strictly necessary or just because the comment above indicated it so?

Reply:
good question. I don't know either. On the other hand we would not want to introduce bugs if we missed relative position needs.

Reply:
Hi @andrewbird and @roelandjansen !  Thanks for taking the time to review my pull request.  I am not entirely sure about the relative position in the code or whether keeping it the same matters.  I looked through the source and based on what I found in the serial number code/demo code later in the file, this is how it removes the demo timer when you activate it with a valid serial number.

Since I am not sure how the serial numbers are generated, and the CD driver didn't appear to come with a serial generator like MOS did, this looked like the best way to do the job without disturbing the existing code.  I also short-circuited the demo message by adding the 13,10,'$' in the data section, so that adds a few bytes to the total size of the driver.

Granted, it could be re-assessed in the future if there is a need to reduce the memory footprint of the driver.  As it stands, the actual vendor CD driver (atapi_cd.sys for example) takes up far more memory.

Reply:
Sure, from my point of view, much less a review and more a question to satisfy my curiosity.